### DISCUSSION TITLE
How to indicates to MudTable Multiselection="True" for example

### DISCUSSION THREAD

**Discussion by gustadelante** [fullstackhero/blazor-wasm-boilerplate#165] [Q&A]
how's it going?
I'm trying to understand the logic of this project. Without problems I was able to create a page based on an entity from the database.

I don't understand **where I could make a MudBlazor table more flexible,** like indicating **MultiSelection** or adding a calculated column or adding a **MudSwitch**...

I am a neweby Any help adds up.
I thank you in advance.

Gus

**Comment by kirkey** [comment]
I am still waiting for someone who could make this up. I wonder why this repository hasn't been maintained enough.

**Comment by neozhu** [comment]
@gustadelante 
you can refer to my project 
https://github.com/neozhu/CleanArchitectureWithBlazorServer

**Reply by tvprasad** [reply]
@gustadelante 
You can find here src/Client/Components/EntityTable
